assets/css/structure-desktop.css

```diff
@@ -0,0 +1,45 @@
+/* Desktop */
+@media (min-width: 901px) {
+  .first-section-container {
+    display: flex;
+    justify-content: space-around;
```

Comment:
justify-content is not the best approach to make columns for structure. You can remove it from here and put the value of each child into their own class. Btw are you sure you don't want to use the grid?